utils/components-utils/src/main/scala/pl/touk/nussknacker/engine/util/parameters/SchemaBasedParameter.scala

```diff
@@ -0,0 +1,49 @@
+package pl.touk.nussknacker.engine.util.parameters
+
+import cats.data.{NonEmptyList, ValidatedNel}
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError
+import pl.touk.nussknacker.engine.api.context.transformation.BaseDefinedParameter
+import pl.touk.nussknacker.engine.api.definition.Parameter
+import pl.touk.nussknacker.engine.api.NodeId
+import pl.touk.nussknacker.engine.util.output.OutputValidatorErrorsConverter
+import pl.touk.nussknacker.engine.util.parameters.SchemaBasedParameter.{ParameterName, RecordFieldName}
+import pl.touk.nussknacker.engine.util.sinkvalue.SinkValueData.TypingResultValidator
```

Comment:
We have to move `TypingResultValidator` to the same package

Reply:
Moved :+1: